repository/src/main/java/tech/ydb/yoj/repository/db/cache/TransactionLocal.java

```diff
@@ -14,12 +16,12 @@
 public final class TransactionLocal {
     private final Map<Supplier<?>, Object> singletons = new IdentityHashMap<>();
 
-    private final Supplier<FirstLevelCache> firstLevelCacheSupplier;
+    private final Supplier<FirstLevelCacheProvider> cacheProvider;
     private final Supplier<ProjectionCache> projectionCacheSupplier;
     private final Supplier<TransactionLog> logSupplier;
 
     public TransactionLocal(@NonNull TxOptions options) {
-        this.firstLevelCacheSupplier = options.isFirstLevelCache() ? FirstLevelCache::create : FirstLevelCache::empty;
+        this.cacheProvider = () -> new FirstLevelCacheProvider(options.isFirstLevelCache() ? FirstLevelCache::create : FirstLevelCache::empty);
```

Comment:
too long line. Please, make creators in separate line or do it like this
```
        this.cacheProvider = () -> new FirstLevelCacheProvider(
                options.isFirstLevelCache() ? FirstLevelCache::create : FirstLevelCache::empty
        );
```

Reply:
:white_check_mark: fixed